newIDE/app/src/Utils/GDevelopServices/Authentication.js

```diff
@@ -59,24 +60,33 @@ export type RegisterForm = {|
   getNewsletterEmail: boolean,
 |};
 
-export type AdditionalUserInfoForm = {|
-  gdevelopUsage?: string,
-  teamOrCompanySize?: string,
-  companyName?: string,
-  creationExperience?: string,
-  creationGoal?: string,
-  hearFrom?: string,
-|};
-
 export type EditForm = {|
-  username: string,
-  description: string,
-  getGameStatsEmail: boolean,
-  getNewsletterEmail: boolean,
-  donateLink: string,
-  communityLinks: CommunityLinks,
+  +username: string,
+  +description: string,
+  +getGameStatsEmail: boolean,
+  +getNewsletterEmail: boolean,
+  +donateLink: string,
+  +communityLinks: CommunityLinks,
```

Comment:
why are those. needed? 
The edit form should always send them no?

Reply:
It's to mark them read only. It was a bit difficult to have Flow satisfied.
It happens because the same function can use either EditFrom, either PatchUserPayload. And to make that work, you need to have those fields read-only.
If we were more accurate, we would add this read-only marker in most of our types